test/Extensions/TesterAzureUtils/AzureCosmosDBMembershipTableTests.cs

```diff
@@ -0,0 +1,102 @@
+using System.Threading.Tasks;
```

Comment:
These tests should go into a different project, something specific to Cosmos DB & the provider. Ideally, there should be a 1:1 relationship between test projects and provider packages.

That's not the case historically, but `TestAzureUtils` is named that way for historical reasons and *should* be split up into multiple smaller test projects (per specific provider).

Reply:
Understood. I'll move all the test to its own project.

Reply:
Done